src/content/developers/docs/gas/index.md

```diff
@@ -21,88 +20,104 @@ Since each Ethereum transaction requires computational resources to execute, eac
 ![A diagram showing where gas is needed in EVM operations](./gas.png)
 _Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
 
-In essence, gas fees are paid in Ethereum's native currency, ether (ETH). Gas prices are denoted in gwei, which itself is a denomination of ETH - each gwei is equal to 0.000000001 ETH (10<sup>-9</sup> ETH). For example, instead of saying that your gas costs 0.000000001 ether, you can say your gas costs 1 gwei.
+Gas fees are paid in Ethereum's native currency, ether (ETH). Gas prices are denoted in gwei, which itself is a denomination of ETH - each gwei is equal to 0.000000001 ETH (10<sup>-9</sup> ETH). For example, instead of saying that your gas costs 0.000000001 ether, you can say your gas costs 1 gwei.
 
-Let's say Alice has to pay Bob 1ETH.
+## Prior to the London upgrade {#pre-london}
+
+The way transaction fees on the Ethereum network are calculated changed during the London network upgrade. Here is a recap of how things used to work:
+
+Let's say Alice had to pay Bob 1 ETH.
 In the transaction the gas limit is 21,000 units and the gas price is 200 gwei.
 
-Total fee will be: `Gas units (limit) * Gas price per unit`
+Total fee would have been: `Gas units (limit) * Gas price per unit`
 i.e `21,000 * 200 = 4,200,000 gwei` or 0.0042 ETH
 
-Now, when Alice sends the money, 1.0042 ETH will be deducted from Alice's account.
-Bob will be credited 1.0000 ETH.
-Miner gets 0.0042 ETH.
+When Alice sent the money, 1.0042 ETH would be deducted from Alice's account.
+Bob would be credited 1.0000 ETH.
+Miner would receive 0.0042 ETH.
 
 This video offers a concise overview of gas and why it exists:
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/AJvzNICwcwc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Why do gas fees exist? {#why-do-gas-fees-exist}
+## After to the London upgrade {#post-london}
 
-In short, gas fees help keep the Ethereum network secure. By requiring a fee for every computation executed on the network, we prevent actors from spamming the network. In order to prevent accidental or hostile infinite loops or other computational wastage in code, each transaction is required to set a limit to how many computational steps of code execution it can use. The fundamental unit of computation is "gas".
+Starting with [the London update](/history/#london), every block has a base fee, the minimum per gas price for inclusion in this block. Optionally, users can set a priority fee (tip).
```

Comment:
I'd suggest adding a brief paragraph to being the "After the London update" where we summarize the high level goals, e.g. 1) reducing variance of tx fees/delays through flexible block sizes, 2) improve UX by making fees more predictable 3) decrease rate of inflation by burning tx fees.

Reply:
Good resource on the "why": https://hackmd.io/@timbeiko/why-1559

Reply:
Hacked.io. Love the idea for getting everyone on the same page.

Reply:
Correction; https://hackmd.io

Reply:
Included this 

```[The London Upgrade](/history/#london) aims to make transacting on Ethereum more predictable for users by overhauling Ethereum's transaction-fee-mechanism. The high-level benefits introduced by this change include better transaction fee estimation, generally quicker transaction inclusion, and reducing the ETH rate of inflation by burning a percentage of transaction fees.```


> Should we also make note EIP-1559 here during the introduction of the London changes?

Thoughts on keeping EIP-1559 discussion contained within the EIP-1559 heading? I could add a small intro saying that most of the changes we're speaking about were introduced in 1559.